### DISCUSSION TITLE
One contract with the same tenderer for different lots - problem with totals of contracts values at notice level (BT-118-NoticeResult)

### DISCUSSION THREAD

**Discussion by alexmohora** [OP-TED/eForms-SDK#703] [Q&A] [answered]
Hello,

We have many situations where contracting authorities conclude one contract for more than one lot if the winner is the same. These situations are mainly in the procurement of pharmaceutical products. There are a lot of lots, usually one per product, but few suppliers. 

They make one contract per supplier and not per lot either for framework agreements or for regular contracts.

For example:
See section V:
https://www.e-licitatie.ro/pub/notices/ca-notices/view-c/100374617
or published on TED, for example contract  Contract nr.: 42307
https://ted.europa.eu/udl?uri=TED:NOTICE:294221-2023:TEXT:RO:HTML&src=0#id3-V.

In this case the ”results of lots” in the eForms award goes something like this:
result 1, lot 31:
   Contract 42307
   estimated value of framework agreement: 1 178 791.20 RON (BT-660-LotResult)
   maxim value of framework agreement: 1 078 826.40 RON (BT-709-LotResult)
result 2, lot 8
   Contract 42307
   estimated value of framework agreement: 1 178 791.20 RON
   maxim value of framework agreement: 1 078 826.40 RON
result 3, lot 13:
  Contract 42307
   estimated value of framework agreement: 1 178 791.20 RON
   maxim value of framework agreement: 1 078 826.40 RON
result 4, lot 16
  Contract 42307
   estimated value of framework agreement: 1 178 791.20 RON
   maxim value of framework agreement: 1 078 826.40 RON

In this case there is a problem with the total value of the contracts BT-118-NoticeResult, BT-1118-NoticeResult (framework agreements) because the CVS validator expect the total to be something like: 1 178 791.20 RON+1 178 791.20 RON+1 178 791.20 RON+1 178 791.20 RON. This is not correct because we don't want to sum the same contract many times.

Please give us some ideas how to proceed in this case!
Thank you
Alex Mohora

**Comment by alexmohora** [comment]
![Screenshot_62](https://github.com/OP-TED/eForms-SDK/assets/115160065/4d22fe4d-aba4-46ad-a461-fb2c5facfaed)
See the situation in this example

**Comment by YvesJo** [comment, marked as answer]
Hi,
Framework values are associated to Lots or Groups of lots individually. It is the same for re-calculated/re-estimated values (associated to Lotresults, i.e. to lots or to Group of lots), they aren't associated directly to the contracts.
Recommendations is to have FA values for the lots and, when an additional value (FA max or FA estimated) is shared amongst lots, also have a group of lots defined with this(these) value(s); The definition of Groups of lots has however to be done already at start of the competition.
